lib/bugspots/scanner.rb

```diff
@@ -14,8 +14,9 @@ def self.scan(repo, branch = "master", depth = 500, regex = nil)
     end
 
     walker = Rugged::Walker.new(repo)
-    walker.sorting(Rugged::SORT_TOPO | Rugged::SORT_REVERSE)
```

Comment:
Yes. 

In `SORT_REVERSE` mode, a walker scans from the initial commit to latest one. I think it would be preferable to scan from latest commit to old one when using `depth` parameter.

I leave a code to specify`Rugged::SORT_TOPO` mode here.
https://github.com/igrigorik/bugspots/pull/35/files#diff-e33b85f28abaec29ab84b822091a6048R17